samples/cloud-shell.yaml

```diff
@@ -20,7 +20,7 @@ spec:
       - type: dockerimage
         memoryLimit: 256Mi
         alias: dev
-        image: 'eclipse/che-remote-plugin-openshift-connector-0.0.17:7.2.0'
+        image: 'quay.io/eclipse/che-sidecar-openshift-connector:0.1.2-2601509'
```

Comment:
```
  - containerID: docker://27efd703e0aeba456e997e75d5ca544b16edc0e46db812242a35626e2fc77d3e
    image: quay.io/eclipse/che-sidecar-openshift-connector:0.1.2-2601509
    imageID: docker-pullable://quay.io/eclipse/che-sidecar-openshift-connector@sha256:8e4403ef1f7f74af3568abd60ccf1b959e2f43bef0001ba7a77ca5f8bc0d5a0e
    lastState:
      terminated:
        containerID: docker://27efd703e0aeba456e997e75d5ca544b16edc0e46db812242a35626e2fc77d3e
        exitCode: 0
        finishedAt: "2020-02-12T10:13:52Z"
        reason: Completed
        startedAt: "2020-02-12T10:13:51Z"
    name: dev
    ready: false
    restartCount: 3
    state:
      waiting:
        message: Back-off 40s restarting failed container=dev pod=workspace2beeb5744d8011ea.workspace-64b6c586dc-vfhk5_default(2c68d46c-4d80-11ea-9885-080027b0607a)
        reason: CrashLoopBackOff
```
Adding
```
        command: ["tail"]
        args: ["-f", "/dev/null"]
```
helped to solve an issue I face

Reply:
For some reason even if dev container is not completed if I did not get my CloudShell workspace on minishift. But using the previous image(`eclipse/che-remote-plugin-openshift-connector-0.0.17:7.2.0`) does not help...
Will continue investigate
![Screenshot_20200212_121932](https://user-images.githubusercontent.com/5887312/74325813-f0933f80-4d91-11ea-82df-771e06b70fc0.png)